model.md

```diff
@@ -5,19 +5,26 @@ Consider the basic scenario of a school looking for a list of suitable and avail
 
 ## The teacher profile
 We can identify the following minimum properties
-* **Essential personal data**
-    * unique identifier, typically an email address or a chose username
-    * First name and surname
-    * Email
-    * Place of residence, possibly only postal code. This is necessary to make a geographical match.
-    * Country. At present littil is only active in the Netherlands, but this can change
-    * Non-essential personal data would be date of birth, gender, telephone, full postal address. It is debatable whether we need at all. The less, the better
-    * Nice to have and without obvious privacy constraints would be a link to a public profile, like a personal website or LinkedIn. A few lines of bio would also be good. 
-* **Teaching skills, preferences and availability**
-  * How experienced is the teacher in giving classes in general? Have they given similar classes before?
-  * What is their area of expertise, with regard to the choice of classes they might provide?
-  * What weekdays are they available?
-  * How far are they willing to travel?
+* unique identifier, generated by the platform. Preferably a UUID, not a DB sequence number.
+* First name
+* Surname
+* Email (must be unique)
+* Postal code, at minimum the digits, not the letters. This is all that is needed for a geographical match.
+* Country. At present littil is only active in the Netherlands, but this can change.
+* Date of last login/activity. Maintained by the platform, not part of the profile.
```

Comment:
Why? Doesn't solutions like Auth0 (which we are looking into if it's a suitable solution for us) keep track of these kind of data? If we don't want so show others the last activity I don't see a reason to keep track on in within the platform itself.